htdocs/core/class/html.form.class.php

```diff
@@ -2344,13 +2344,13 @@ function select_produits_fournisseurs_list($socid,$selected='',$htmlname='produc
      *  @param      string	$htmlname        Name of HTML field
      *  @return		string|null
      */
-    function select_product_fourn_price($productid,$htmlname='productfournpriceid')
+    function select_product_fourn_price($productid, $selected_supplier='', $htmlname='productfournpriceid')
```

Comment:
@eldy No problem but i did that because i only found 1 call of this function.
As you want, i corrected it to keep params order.